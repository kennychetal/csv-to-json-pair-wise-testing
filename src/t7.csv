letters,numbers,notsymbols,
a,4,@,
b,7,$,
c,5,%,