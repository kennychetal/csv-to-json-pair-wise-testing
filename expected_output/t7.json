[
    {
        "notsymbols": "@", 
        "letters": "a", 
        "numbers": "4"
    }, 
    {
        "notsymbols": "$", 
        "letters": "b", 
        "numbers": "7"
    }, 
    {
        "notsymbols": "%", 
        "letters": "c", 
        "numbers": "5"
    }
]